sdk/scheduler/src/main/java/com/mesosphere/sdk/api/PlansResource.java

```diff
@@ -167,7 +163,7 @@ public Response continueCommand(
             }
 
             boolean allInProgress = phases.stream()
-                    .filter(phz -> phz.isInProgress())
+                    .filter(phz -> phz.isRunning())
```

Comment:
Minor: `phz` -> `phase`.

Reply:
I had meant to change this as well, but it conflicts with the `String phase` method param

Reply:
OK. Didn't see that. In this case one could use:
```java
boolean allInProgress = phases.stream()
                                                    .filter(Phase::isRunning)
```
For that matter, looking at the code around the diff again, one could also use `allMatch` here instead of comparing the size of the filtered stream to the original list.